SYCL/lit.cfg.py

```diff
@@ -313,6 +313,19 @@
 if find_executable('sycl-ls'):
     config.available_features.add('sycl-ls')
 
+# TODO properly set XPTIFW include and runtime dirs
```

Comment:
The change looks like a hack.
We can detect the presence of the library in Cmaake and pass this value through lit.site.cfg.py. Also, you can get proper include and other directories from CMake configuration to resolve TODO above.